### PR TITLE
Tracking Plan Ids

### PR DESCRIPTION
**Pull request by jfoskin**
### Proposed changes

Provide customers with information about the two tracking plan ids they might encounter

### CODE REVIEW

File: src/protocols/faq.md [hunk 1]
```diff
@@ -84,6 +84,15 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
...
 
+### Why do I have to different Tracking Plan Ids?
+
+When you access a Tracking Plan, you'll come across two IDs: tp_ and rs_:
+
+If you are using Public API, copy the ID that starts with "tp". 
+
+If you are using Config API, copy the ID that starts with "rs". 
+
+*Please be aware that depending on how you access your tracking plan will depend on the slug that is presented in the URL.*
```
Comment (forstisabella) [inline src/protocols/faq.md:95]: What does the difference look like? If you access it in the app it has one URL, and if you access it using the API it has a different URL?

**Review by forstisabella — CHANGES_REQUESTED** [review]

**Review by forstisabella — APPROVED** [review]

File: src/protocols/faq.md [hunk 2]
```diff
@@ -82,11 +82,20 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
 
 Unplanned property omission is only supported for cloud-mode destinations. Unplanned properties will not be omitted when they're sent to device-mode destinations.
 
+### Why do I have to different Tracking Plan IDs?
```
Comment (forstisabella) [inline src/protocols/faq.md:85]:
```suggestion
### Why do I have two different Tracking Plan IDs?
```

File: src/protocols/faq.md [hunk 3]
```diff
@@ -82,11 +82,20 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
...
+
+When you access a Tracking Plan, you'll come across two IDs: `tp_` and `rs_`.
+
+If you are using Public API, copy the ID that starts with "tp". 
+
+If you are using Config API, copy the ID that starts with "rs". 
+
+*Please be aware that the way you access your Tracking Plan determines the slug that is present in the URL.*
```
Comment (forstisabella) [inline src/protocols/faq.md:93]: @jfoskin - when you say "slug", are you referring to the prefix (`tp_` or `rs_`) of your tracking plan ID?

**Comment by cmastr** [comment]
requested answers via Slack

File: src/protocols/faq.md [hunk 4]
```diff
@@ -85,10 +85,25 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
...
 
+### Why do I have two different Tracking Plan IDs?
+
+When you access a Tracking Plan, you'll come across two IDs: `tp_` and `rs_`.
+
+If you are using Public API, copy the ID that starts with "tp". 
+
+If you are using Config API, copy the ID that starts with "rs". 
+
+*Please be aware that the way you access your Tracking Plan determines the slug that is present in the URL.*
```
Comment (forstisabella) [inline src/protocols/faq.md:96]:
```suggestion
```

File: src/protocols/faq.md [hunk 5]
```diff
@@ -85,10 +85,25 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
 
 Unplanned property omission is only supported for cloud-mode destinations. Unplanned properties will not be omitted when they're sent to device-mode destinations.
 
+### Why do I have two different Tracking Plan IDs?
+
+When you access a Tracking Plan, you'll come across two IDs: `tp_` and `rs_`.
+
+If you are using Public API, copy the ID that starts with "tp". 
+
+If you are using Config API, copy the ID that starts with "rs".
```
Comment (forstisabella) [inline src/protocols/faq.md:94]:
```suggestion
If you're using the Config API, you'll need the ID that starts with `rs`. 
```

File: src/protocols/faq.md [hunk 6]
```diff
@@ -85,10 +85,25 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
 
 Unplanned property omission is only supported for cloud-mode destinations. Unplanned properties will not be omitted when they're sent to device-mode destinations.
 
+### Why do I have two different Tracking Plan IDs?
+
+When you access a Tracking Plan, you'll come across two IDs: `tp_` and `rs_`.
```
Comment (forstisabella) [inline src/protocols/faq.md:90]:
```suggestion
When you access a Tracking Plan, you'll come across two IDs: `tp_` and `rs_`. Segment uses the two IDs to identify your Tracking Plan in the two APIs you can use to manage your workspace: the [Public API](/docs/api/public-api/) and the [Config API](/docs/api/config-api/). 

To view the two IDs for your Tracking Plan, navigate to the Tracking Plan you'd like to view the ID for and select the dropdown next to **Tracking Plan ID**. 
```

File: src/protocols/faq.md [hunk 7]
```diff
@@ -85,10 +85,25 @@ Segment's code uses built-in logic to verify if an event exists in the Tracking
...
+
+If you are using Public API, copy the ID that starts with "tp".
```
Comment (forstisabella) [inline src/protocols/faq.md:92]:
```suggestion
If you're using the Public API, you'll need the ID that starts with `tp_`. 
```

**Review by forstisabella — APPROVED** [review]

**Pull request merged**